Template/Variable/EtrackerConfigurationVariable.php

```diff
@@ -0,0 +1,126 @@
+                $field->title = 'Variable et_pagename';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_areas', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_areas';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_target', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_target';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_tval', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_tval';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_tonr', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_tonr';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_tsale', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_tsale';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_basket', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_basket';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('et_cust', '', FieldConfig::TYPE_STRING, function (FieldConfig $field) {
+                $field->title = 'Variable et_cust';
+                $field->customUiControlTemplateFile = self::FIELD_TEMPLATE_VARIABLE;
+                $field->description = 'Set variable if you want to overwrite the default value';
+            }),
+            $this->makeSetting('customDimensions', array(), FieldConfig::TYPE_ARRAY, function (FieldConfig $field) {
```

Comment:
Just double checking, should this custom dimensions setting be removed maybe? or does etracker support custom dimensions through this property?

Reply:
Custom dimensions are possible to define, so we'd like to implement it as well.

Reply:
sweet, just wanted to make sure it'll be implemented and wasn't a copy/paste error